bin/validate-workflow-tags.sh

```diff
@@ -31,7 +34,7 @@ do
                 echo "Checking tutorial $w for tags" 
                 if tester $w $topic;
                 then
-                    echo "ERROR: No 'tags' attribute found with its corresponding topic or workflow includes tool(s) from testtoolshed."
+                    echo "ERROR: No 'tags' attribute found with its corresponding topic or 'annotation' attribute with for example the title of the tutorial or workflow includes tool(s) from testtoolshed."
```

Comment:
you might consider:

1. in the tester script write to sys.stderr with the message that's more clear/detailed ("in wf X an annotation is missing, it should be Y")
2. Not writing an error in the bash portion and just setting the overall exit code

Reply:
don't know 100% what you mean with  just setting the overall exit code, but I've added sys.stderr codes. Thanks for the suggestion.

Reply:
The changes you made look good :)